### PR TITLE
Release 1.13.2

### PR DESCRIPTION
**Pull request by stevehobbsdev**
Add support for token validation for Organizations [\#631](https://github.com/auth0/auth0-spa-js/pull/631) ([stevehobbsdev](https://github.com/stevehobbsdev))

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+## [v1.13.2](https://github.com/auth0/auth0-spa-js/tree/v1.13.2) (2020-11-09)
+
+[Full Changelog](https://github.com/auth0/auth0-spa-js/compare/v1.13.1...v1.13.2)
+
+**Added**
+
+- [SDK-2121] Add support for token validation for Organizations [\#631](https://github.com/auth0/auth0-spa-js/pull/631) ([stevehobbsdev](https://github.com/stevehobbsdev))
```
Comment (frederikprijck) [inline CHANGELOG.md:9]: Shouldnt this be 1.14.0 ?

File: docs/globals.html [hunk 2]
```diff
@@ -2799,8 +2799,10 @@ <h1>@auth0/auth0-spa-js</h1>
 			</a>
 			<p>Auth0 SDK for Single Page Applications using <a href="https://auth0.com/docs/api-auth/tutorials/authorization-code-grant-pkce">Authorization Code Grant Flow with PKCE</a>.</p>
 			<p><a href="https://circleci.com/gh/auth0/auth0-spa-js"><img src="https://circleci.com/gh/auth0/auth0-spa-js.svg?style=svg" alt="CircleCI"></a>
-				<a href="https://opensource.org/licenses/MIT"><img src="https://img.shields.io/:license-mit-blue.svg?style=flat" alt="License"></a>
-			<a href="https://app.fossa.com/projects/git%2Bgithub.com%2Fauth0%2Fauth0-spa-js?ref=badge_shield"><img src="https://app.fossa.com/api/projects/git%2Bgithub.com%2Fauth0%2Fauth0-spa-js.svg?type=shield" alt="FOSSA Status"></a></p>
+				<img src="https://img.shields.io/github/v/release/auth0/auth0-spa-js" alt="Release">
+				<a href="https://codecov.io/gh/auth0/auth0-spa-js"><img src="https://img.shields.io/codecov/c/github/auth0/auth0-spa-js" alt="Codecov"></a>
+				<img src="https://img.shields.io/npm/dw/@auth0/auth0-spa-js" alt="Downloads">
+			<a href="https://opensource.org/licenses/MIT"><img src="https://img.shields.io/:license-mit-blue.svg?style=flat" alt="License"></a></p>
```
Comment (frederikprijck) [inline docs/globals.html:2805]: Is this change intended?

**Review by frederikprijck — APPROVED** [review]
Looks good, just two questions.

File: docs/globals.html (hunk 2 above)
Comment (stevehobbsdev) [reply docs/globals.html:2805]: Yeah I think this is just rebuilding the readme for docs based on #620

File: CHANGELOG.md (hunk 1 above)
Comment (stevehobbsdev) [reply CHANGELOG.md:9]: As it's an internal non-breaking change without any public API changes, I figured a patch version would be more appropriate but happy to hear otherwise.

Comment (frederikprijck) [reply CHANGELOG.md:9]: Sure, I am fine with either. Just wanted to ensure this was intended.
